src/docs/releases/2.1.0.md

```diff
@@ -50,6 +50,15 @@ The following properties of `RegistrationSettings` are now deprecated and will b
 
 Previously, the `UsersCanRegister` property controlled which types of registration were allowed. With this update, this property is obsolete and will be removed in a future release. To enable site registration now, simply activate the **User Registration** feature.
 
+### Roles Feature
+
+In the Roles feature, there were previously `AssignRoles` and `AssignRole_{RoleName}` permissions, alongside `AssignRoleToUsers` and `AssignRoleToUsers_{RoleName}` permissions. As these permissions were redundant, `AssignRoles` and `AssignRole_{RoleName}` have been removed in favor of the latter.
+
+To ensure smooth transitions, a migration step has been added. Any roles previously granted `AssignRoles` will now be automatically granted `AssignRoleToUsers`. Similarly, any roles with `AssignRole_{RoleName}` will now be granted the corresponding `AssignRoleToUsers_{RoleName}` permission.
+
+!!! note
```

Comment:
This is actually also better for the other note. See https://squidfunk.github.io/mkdocs-material/reference/admonitions/.

```suggestion
!!! warning
```